test/intfuncs.jl

```diff
@@ -35,6 +35,31 @@
         @test_throws OverflowError lcm(typemin(T), typemin(T))
     end
 end
+@testset "gcd/lcm for arrays" begin
+    for T in (Int32, Int64)
+        @test gcd(T[]) === T(0)
+        @test gcd(T[3, 5]) === T(1)
+        @test gcd(T[3, 15]) === T(3)
+        @test gcd(T[0, 15]) === T(15)
+        @test gcd(T[3,-15]) === T(3)
+        @test gcd(T[-3,-15]) === T(3)
+        @test gcd(T(0, 0)) === T(0)
```

Comment:
Should be `T[0, 0]`.

Reply:
Thanks to you (and to Travis :) ) for catching that! Just pushed a fix.